internal/providers/msentraid/himmelblau.go

```diff
@@ -0,0 +1,188 @@
+package msentraid
+
+//go:generate ./generate-himmelblau.sh
+
+/*
+#cgo LDFLAGS: -L${SRCDIR} -Wl,-rpath=${SRCDIR} -lhimmelblau
+#include "himmelblau.h"
+*/
+import "C"
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"unsafe"
+
+	"github.com/ubuntu/authd/log"
+	"golang.org/x/oauth2"
+)
+
+var tpm *C.BoxedDynTpm
+var authValue *C.char
+var loadableMachineKey *C.LoadableMachineKey
+var machineKey *C.MachineKey
+var brokerClientApp *C.BrokerClientApplication
+
+func init() {
```

Comment:
Wouldn't be better to move this to a init version that is controlled manually and we can handle the error for?